browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -79,7 +79,17 @@ public BrowserSwitchPendingRequest start(@NonNull ComponentActivity activity, @N
         }
     }
 
-    void assertCanPerformBrowserSwitch(ComponentActivity activity, BrowserSwitchOptions browserSwitchOptions) throws BrowserSwitchException {
+    /**
+     * Throws a {@link BrowserSwitchException} when a browser switch flow cannot be started.
+     *
+     * @param activity the activity used to start browser switch
+     * @param browserSwitchOptions {@link BrowserSwitchOptions} the options used to configure the browser switch
+     * @throws BrowserSwitchException exception containing the error message on why browser switch cannot be started
+     */
+    public void assertCanPerformBrowserSwitch(
```

Comment:
Is restricting to library scope similar to what we are doing in the core SDK in v5 not possible here?

Reply:
Unfortunately, the restrict to annotation wouldn't work here as it'd be tied to the Gradle group ID which is different between browser switch and Venmo. The PayPal module also uses this function.

It also might make sense to make public to other consumers of Browser Switch. Since we're using the function within the BT SDK, other consumers might want to access it as well.